environments/common/rds/rds.tf

```
@@ -51,3 +52,13 @@ resource "random_string" "prefix" {
   special = false
   numeric = false
 }
+
+# rds private subnet
+resource "aws_db_subnet_group" "rds_private_subnet" {
+  name       = "rds-subnet-group-${var.environment}"
+  subnet_ids = var.private_subnets_id
+
+  tags = {
+    Name = "RBS Private Subnet Group ${var.environment}"
```

```suggestion
    Name = "RDS Private Subnet Group ${var.environment}"
```

Just for consistency :)